test/commons/dom/get-element-stack.js

```diff
@@ -206,34 +206,56 @@ describe('dom.getElementStack', function () {
         '</div>' +
         '</div>';
       axe.testUtils.flatTreeSetup(fixture);
-      var target = fixture.querySelector('#target');
-      var stack = mapToIDs(getElementStack(target));
+      const target = fixture.querySelector('#target');
+      const stack = mapToIDs(getElementStack(target));
       assert.deepEqual(stack, ['1', '4', 'target', '5', '3', '2']);
     });
 
-    it('should correctly order children of position elements without z-index', function () {
+    it('should correctly order children of position elements without z-index', () => {
       fixture.innerHTML =
         '<div id="1" style="position:relative;width:40px;height:40px;">' +
         '<div id="target" style="width:40px;height:40px;"></div>' +
         '</div>';
       axe.testUtils.flatTreeSetup(fixture);
-      var target = fixture.querySelector('#target');
-      var stack = mapToIDs(getElementStack(target));
+      const target = fixture.querySelector('#target');
+      const stack = mapToIDs(getElementStack(target));
       assert.deepEqual(stack, ['target', '1', 'fixture']);
     });
 
-    it('should correctly order children of position elements with z-index', function () {
+    it('should correctly order children of position elements with z-index', () => {
       fixture.innerHTML =
         '<div id="1" style="position:relative;width:40px;height:40px;z-index:1">' +
         '<div id="target" style="width:40px;height:40px;"></div>' +
         '</div>';
       axe.testUtils.flatTreeSetup(fixture);
-      var target = fixture.querySelector('#target');
-      var stack = mapToIDs(getElementStack(target));
+      const target = fixture.querySelector('#target');
+      const stack = mapToIDs(getElementStack(target));
       assert.deepEqual(stack, ['target', '1', 'fixture']);
     });
 
-    it('should handle modals on top of the stack', function () {
+    [
+      'flex',
+      'inline-flex',
+      'inline flex',
+      'grid',
+      'inline-grid',
+      'inline grid'
+    ].forEach(type => {
+      it(`should correctly order "${type}" items with z-index`, () => {
+        fixture.innerHTML = `
+          <div id="1" style="position:absolute;width:40px;height:40px;z-index:1"></div>
+          <div id="2" style="display: flex">
```

Comment:
I think you mean this?

```suggestion
          <div id="2" style="display: ${type}">
```

Reply:
Good catch!